src/workers/hermes_strategies/upload_strategy.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+export default class HermesUploadStrategy {
+  bundlingPeriod() {
+    return 1000;
+  }
+
+  storagePeriods() {
+    return 2;
+  }
+
+  async shouldBundle() {
```

Comment:
decideIfShouldBundle?

Reply:
Imo shouldBundle is ok. Simple descriptive question and yes/no answer.